ocfweb/docs/docs/services/account/content-removal.md

```diff
@@ -0,0 +1,63 @@
+[[!meta title="Content Removal Policy"]]
+
+
+## Introduction
+
+Most of the content on [www.ocf.berkeley.edu](https://www.ocf.berkeley.edu)
+is created and owned by OCF members, not the OCF itself. This user-generated
+content is not endorsed or supported by the OCF nor is the OCF responsible
+for content posted by OCF members.
+
+The OCF maintains a strict policy not to remove or modify data
+hosted by members unless legally compelled to do so, or if such
+data is in violation of existing OCF
+[[account policies|doc services/account/account-policies]].
+
+This means the OCF will not change or remove any text, pictures,
+information, or other data hosted by an OCF member. We do however
+reserve the right to deactivate inactive accounts at our discretion.
+
+## Suggestions
+
+If you take issue with the content hosted on the website of an OCF user,
+the responsibility falls on you to resolve any conflict.
+
+We expect you to make a serious attempt to contact the owner of
+the content in question and request that they address your concern
+directly. You may try to find their contact information on their
+website or in the [University directory](//directory.berkeley.edu).
+If this fails, you may try to contact the owner by the contact email
+associated with their OCF account. If the website in question is
+`https://www.ocf.berkeley.edu/~username`, the contact address will be
+<username>@ocf.berkeley.edu.
+
+If repeated attempts to contact the owner of the account have failed,
+you may [[contact us|doc contact]] and request the website be
+disabled due to inactivity. If we attempt to contact the website owner
+and fail to get a response, we may disable their account as per our
+[[account policies|doc services/account/account-policies]].
+
+However, if the website owner responds and refuses your removal request,
+the OCF cannot take any further action on your behalf unless the website
+is in violation of our
+[[account policies|doc services/account/account-policies]]. If you believe
+this is the case, you may notify us of the exact policy you believe to have
+been breached, and if we determine the complaint to be valid, we will take
+appropriate action on our end.
+
+Please note that, if the above steps fail, the website is being maintained
+in compliance with our policies. There is nothing the OCF can do to change
```

Comment:
I dunno. If the owner's website weren't compliant with OCF policy, wouldn't our hypothetical plaintiff report it to us after reading the previous paragraph?

Probably we should just change it to "likely being maintained".